spec/models/bet_spec.rb

```diff
@@ -11,11 +11,14 @@
     User.create(username: 'Jack', email: 'jack@mail.com', password: 'notjack')
   end
 
-  let(:bet1) do
-    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!')
+  let!(:bet1) do
+    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!',
+               expires_at: DateTime.current + 1, creator_id: bob.id)
+  end
+  let!(:option1) { BetOption.create(option_text: 'Atlanta wins', bet: bet1) }
```

Comment:
I dont think these need to be bang methods.... Can you explain why you think you need them?